test/e2e/kubectl/kubectl.go

```diff
@@ -1072,7 +1072,7 @@ metadata:
 			})
 			validateService := func(name string, servicePort int, timeout time.Duration) {
 				err := wait.Poll(framework.Poll, timeout, func() (bool, error) {
-					endpoints, err := c.CoreV1().Endpoints(ns).Get(name, metav1.GetOptions{})
+					ep, err := c.CoreV1().Endpoints(ns).Get(name, metav1.GetOptions{})
 					if err != nil {
```

Comment:
Rename variable `endpoints` to `ep` because we need to import `endpoints` package in this file